### DISCUSSION TITLE
Two grids in a grid but different height? How?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#5987] [Help] [answered]
Can I ask you for help with this https://play.tailwindcss.com/fIuk5aXhz0?

- What I have now:

![image](https://user-images.githubusercontent.com/41120635/140434909-135130d7-86f6-4258-bfb9-e88d58c5ae41.png)

- What I need here is to have the red div height lower, just after `HELLO 2` of the green box, like this:

![image](https://user-images.githubusercontent.com/41120635/140435068-7536ca7d-0f13-43b0-91b3-8ce937ede6c7.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

You can add [the `items-start` class](https://tailwindcss.com/docs/align-items#start) to the parent grid, so the red nested grid doesn't stretch all the way down to match the green grid's height:

https://play.tailwindcss.com/gLdhP62UZT

![image](https://user-images.githubusercontent.com/485747/140484670-075ffc5f-1503-48c1-85a8-4a6c8050274e.png)

![image](https://user-images.githubusercontent.com/485747/140484908-b0e16975-2724-4226-85d7-f37524643e77.png)

**Reply by frederikhors** [reply]
@simonswiss thanks for your answer.

What do you think about this instead https://play.tailwindcss.com/baaOhBg41s?

**Reply by simonswiss** [reply]
Looks like adding a separating div between the two grids works too.

I'd say using `items-start` here is the more "correct" way to handle it, but nothing will go fundamentally wrong with an extra div wrapper in your markup.

**Comment by andcl** [comment]
Hey, so... How would you insert a third div right below the red one without leaving the space that is generated to align the new one with the green div? Any ideas?
